cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -153,16 +153,16 @@ func RelayBatchProphecyCompletedToEthereum(
 	// sleep 2 seconds to wait for tx to go through before querying.
 	sleepThread(2)
 
-	if err != nil {
-		sugaredLogger.Errorw(
-			"cosmosBridgeInstance.BatchSubmitProphecyClaimAggregatedSigs",
-			"batchID", batchID,
-			"batchClaimData", batchClaimData,
-			"batchSignatureData", batchSignatureData,
-			errorMessageKey, err,
-		)
-		return err
-	}
+	// if err != nil {
```

Comment:
This if statement can't come out; we'll crash when we try to dereference tx later on

Reply:
sorry, I am still debuging some issues. I will set the pr ready for review after test passed.